code/modules/admin/debug_verbs.dm

```diff
@@ -438,7 +438,7 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 	set name = "View Runtimes"
 	set desc = "Open the runtime Viewer"
 
-	if(!check_rights(R_DEBUG))
+	if(!check_rights(R_RUNTIME|R_DEBUG))
 		return
```

Comment:
Does this not check both ie shouldn't this be two check rights checks

Reply:
I had the same concern, but it does indeed check for any like:

https://github.com/tgstation/TerraGov-Marine-Corps/blob/9fe6280ef69d7144135b281f26d8563ba454aaaa/code/modules/admin/admin_verbs.dm#L496